graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/ScalarsDmlTest.java

```diff
@@ -192,6 +193,7 @@ public void uuidTypeShouldSupportUuidFunction() {
         .isEqualTo(4);
   }
 
+  @SuppressWarnings("unused")
```

Comment:
How comes those are unused, yet useful? Out of curiosity.

Reply:
They provide the parameters to the test methods, see the `@MethodSource` annotations. I will add an explanatory comment.